kobo/apps/superuser_stats/templates/superuser_stats.html

```diff
@@ -0,0 +1,35 @@
+{% load i18n %}
+
+{% block content_title %}
+    <h1>Superuser Stats</h1>
+{% endblock %}
+
+{% block table_list %}
+    <table>
+      <thead>
+        <tr>
+          <th>
+            Reports
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>
+            <form method="GET" action="{{ kpi_base_url }}/superuser_stats/country_report/">
+              <label for="start_date">Start Date</label>
+              <input type="date" id="start_date" name="start_date">
+              <label for="end_date">End Date</label>
+              <input type="date" id="end_date" name="end_date">
+              <button>Submit</button>
+            </form>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <a href="{{ kpi_base_url }}/superuser_stats/user_report/">User Report</a>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+{% endblock %}
```

Comment:
I would move this template under `usage_statistics` app.

Reply:
It seems to have an issue with the CSS of the table. It takes 100% of the width. If the viewport width is too small, the count column is hidden by the filter. 

![Screen Shot 2021-06-22 at 10 13 51](https://user-images.githubusercontent.com/24226825/122940814-ce74de80-d342-11eb-8413-124c6c52429c.png)
![Screen Shot 2021-06-22 at 10 14 00](https://user-images.githubusercontent.com/24226825/122940822-d0d73880-d342-11eb-9f01-6afb0b276651.png)

This css rule fixes it. I don't know if you can change the CSS directly or if you need to include a stylesheet that override the behaviour for this module. Last solution, you can add the style within the template.

```css
#changelist table {
    width: calc(100% - 250px);
}
```